packages/docusaurus-theme-classic/src/theme/MDXComponents/index.js

```diff
@@ -16,6 +16,9 @@ export default {
   code: (props) => {
```

Comment:
I wonder why we even map code to CodeBlock in the first place. Endi did this in the first commit of this file. Any idea?

Reply:
Nope :man_shrugging:

Reply:
This is what permits to transform ``` to the prism editor